wordpress-multi-server.yaml

```diff
@@ -1,8 +1,12 @@
 heat_template_version: 2013-05-23
 
 description: |
-  This is a Heat template to deploy Load Balanced WordPress servers with a
-  backend database server.
+  Multiple [WordPress](https://wordpress.org/about) v3.9.2 servers with
```

Comment:
We'd also like for all of these descriptions to have heading's too:
```
#### Multiple Servers
```
which when parsed looks like:
#### Multiple Servers